server/config_check_test.go

```diff
@@ -1113,6 +1113,72 @@ func TestConfigCheck(t *testing.T) {
 			errorLine: 0,
 			errorPos:  0,
 		},
+		{
+			name: "when setting latency tracking without a system account",
+			config: `
+                # system_account: sys
+
+                accounts {
+                  sys {
+                   users = [
+                     {user: sys, pass: "" }
+                   ]
+                  }
+                  nats.io: {
+                    users = [
+                      { user : bar, pass: "" }
+                    ]
+
+                    exports = [
+                      { service: "nats.add" 
+                        response: singleton # chunked
+                        latency: {
+                          sampling: 100%
+                          subject: "latency.tracking.add"
+                        }
+                      }
+
+                    ]
+                  }
+                }
+                `,
+			err:       errors.New(`Error adding service latency sampling for "nats.add": system account not setup`),
+			errorLine: 4,
+			errorPos:  17,
+		},
+		{
+			name: "when setting latency tracking with a system account",
+			config: `
+                system_account: sys
+
+                accounts {
+                  sys {
+                   users = [
+                     {user: sys, pass: "" }
+                   ]
+                  }
+                  nats.io: {
+                    users = [
+                      { user : bar, pass: "" }
+                    ]
+
+                    exports = [
+                      { service: "nats.add" 
+                        response: singleton # chunked
+                        latency: {
+                          sampling: 100%
+                          subject: "latency.tracking.add"
```

Comment:
when we try to use a wildcard subject here like `latency.results.*` we get an error like: 

```
nats-server: examples/latency-example.conf:4:1: Error adding service latency sampling for "nats.add": invalid publish subject
```

I think we should mention the subject to clarify that the bad subject is the one used for the latency samples

Reply:
Yes it needs to be a valid publish subject, meaning literal.